app/src/controllers/object.js

```diff
@@ -391,6 +470,41 @@ const controller = {
     }
   },
 
+  /**
+   * @function replaceTags
+   * Replaces the tag set on the requested object
+   * @param {object} req Express request object
+   * @param {object} res Express response object
+   * @param {function} next The next callback function
+   * @returns {function} Express middleware function
+  */
```

Comment:
General JSDoc indentation spacing seems a bit off in this file. Double check when you can.